CONTRIBUTING.md

```diff
@@ -62,28 +62,34 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Collaborator
+# Becoming a Member and Collaborator
 
-All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, 
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by 
+filing an issue. Once added as an Observer to meetings, we will track attendance and participation 
```

Comment:
A bot would be awesome! And yes, we do already have the information needed to make these assessments, it's just a really manual process right now. I'll also add that figuring out timing right now is a bit tricky since all timing can't be retroactive. For the time being, until we hit three months post this PR landing (and two months for the attendance requirement for members to become active), it's especially tricky. In time, this will all smooth out though!